### PR TITLE
New Uzzinivin Spice

### PR DESCRIPTION
**Pull request by ytteroy**
## Description of new Instant Answer, or changes

Get car VIN from registration number in Latvia 

## People to notify

---

Instant Answer Page: https://duck.co/ia/view/uzzinivin

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/spice/uzzinivin/uzzinivin.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.zci--uzzinivin {
```
Comment (pjhampton) [inline share/spice/uzzinivin/uzzinivin.css]: You can remove this file if there is no styles

File: share/spice/uzzinivin/uzzinivin.handlebars [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<h3 class="c-base__title text--primary">
+  {{#if this.found}}
+    <a href="https://uzzinivin.lv/v/{{this.vin}}">{{this.vin}}</a>
+  {{else}}
+    <span style="color:#de5833;">VIN not found. Try again later.</span>
```
Comment (J-tt) [inline share/spice/uzzinivin/uzzinivin.handlebars]: Shouldn't this be in same language as the search term?

File: share/spice/uzzinivin/uzzinivin.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          templates: {
+                group: 'text',
+                options: {
+                    title_content: Spice.uzzinivin.uzzinivin,
+                    moreAt: true
+                }
+            }
+        });
+    };
+}(this));
```
Comment (J-tt) [inline share/spice/uzzinivin/uzzinivin.js]: No newline at end of file

File: share/spice/uzzinivin/uzzinivin.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        if (!api_result || !api_result.found) {
+            return Spice.failed('uzzinivin');
+        }
+
+        // Render the response
+        Spice.add({
+            id: 'uzzinivin',
+
+            // Customize these properties
+            name: 'UzziniVIN',
```
Comment (J-tt) [inline share/spice/uzzinivin/uzzinivin.js]: 'Uzzini VIN' is probably more user friendly

**Comment by pjhampton** [comment]
Hey @ytteroy 

[DuckDuckHack is now in Maintenance Mode](https://duckduckhack.com/) and from now on, we are only accepting issues and PRs for essential bugs and bug fixes.

Unfortunately, this is non-essential and will be closed.

We appreciate you taking the time to contribute and apologize for not being able to include your work.

**Pull request closed without merging**